client/securedrop_client/crypto.py

```diff
@@ -86,7 +87,7 @@ def __init__(self, sdc_home: str, session_maker: scoped_session, is_qubes: bool)
         self.is_qubes = is_qubes
         self.session_maker = session_maker
 
-        config = Config.from_home_dir(self.sdc_home)
+        config: Any = Config.load()
```

Comment:
@legoktm, do you see a way around the `Any` annotation here if `Config` is populated via `setattr()`?

Reply:
Done in 27bcc06fa35b7c1f826b441ac9a5c34477f6909e.